src/test/regress/expected/bfv_partition.out

```diff
@@ -1938,3 +1938,116 @@ DROP TABLE part_tbl;
 -- start_ignore
 drop schema if exists bfv_partition;
 -- end_ignore
+---
```

Comment:
I think test addition after cleanup (line 1937) is a bad idea. This test set use `set search_path=bfv_partition;`. As result threre are failed tests with:

```
ERROR:  no schema has been selected to create in
```

I'm going to attach the test output to Jira issue.

Reply:
Agreed, fixed below.